### DISCUSSION TITLE
SpotX-Cli

### DISCUSSION THREAD

**Discussion by amd64fox** [SpotX-Official/SpotX#435] [General]
Hi @SudoVanilla

I will reply to your [message](https://github.com/SpotX-CLI/SpotX-Linux/pull/32#issuecomment-1575881729) here

This [org](https://github.com/SpotX-CLI) is dead, the people who were working on the code have left the organization. You can switch to the new repository for [Linux and Mac](https://github.com/jetfir3/SpotX-Bash), which is being maintained by @jetfir3.

Cheers.

**Comment by jetfir3** [comment]
I was originally part of the CLI org and maintained the macOS/Linux repos but have since switched to a single repo as mentioned above. 

All Spotify installations on Linux should be auto-detected by [SpotX-Bash](https://github.com/jetfir3/SpotX-Bash) and not require additional instructions per packager manager. The `-P` flag is still available for special cases or if current detection were to fail in a future client update before SpotX-Bash has been updated. 

Additionally, we provide pre-patched Snap installations in our Telegram discussion community for those who use/prefer Snap rather than Flatpak.
